releases/8.0.0.md

```diff
@@ -135,6 +138,7 @@ JavaScript:
 
 - The format of JavaScript files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
 - The format of how some images are loaded in template files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
+- Update any imports from `@primevue/themes` to `@primeuix/themes`.
```

Comment:
@johnatawnclementawn can you spot check if lingo looks reasonable after making this change? 🙏